src/contrib/mlir/core/ngraph_dialect/fused_ops.td

```diff
@@ -32,14 +33,899 @@ def NGSqueezeOp :
+    Arguments<(ins NG_TensorType:$data, F64Attr:$min, F64Attr:$max)>
+{
+  let summary = "Clamp Op";
+  let description = [{
+    Performs a clipping operation on all elements of the input node
     
+    All input values that are outside of the <min;max> range are set to 'min' or 'max'
+    depending on which side of the <min;max> range they are. The values that fall into
+    this range remain unchanged.
+  }];
+
+  let extraClassDeclaration = [{
+    void setMin(const Attribute& attr) { this->setAttr("min", attr); }
+    void setMax(const Attribute& attr) { this->setAttr("max", attr); }
+  }];
+}
+
+// Gelu Op
+def NGGeluOp :
+    NG_OneResult_Op<"gelu", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data)>
+{
+  let summary = "Gelu Op";
+  let description = [{
+    Gaussian Error Linear Unit
+    f(x) = 0.5 * x * (1 + erf( x / sqrt(2) )
+  }];
+}
+
+// GeluBackpropFactor Op
+def NGGeluBackpropFactorOp :
+    NG_OneResult_Op<"geluBackpropFactor", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data)>
+{
+  let summary = "Gelu Backprop Op";
+  let description = [{
+    Backprop for Gelu(x) is GeluBackprop(x) * delta
+  }];
+}
+
+// Elu Op
+def NGEluOp :
+    NG_OneResult_Op<"elu", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data, F64Attr:$alpha)>
+{
+  let summary = "Elu Op";
+  let description = [{
+    Exponential Linear Unit
+    x <  0 => f(x) = alpha * (exp(x) - 1.)
+    x >= 0 => f(x) = x
+  }];
+
+  let extraClassDeclaration = [{
+    void setAlpha(const Attribute& attr) { this->setAttr("alpha", attr); }
+  }];
+}
+
+// FakeQuant Op
+def NGFakeQuantOp :
+    NG_OneResult_Op<"fakeQuant", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data, NG_TensorType:$inputLow, NG_TensorType:$inputHigh, 
+               NG_TensorType:$outputLow, NG_TensorType:$outputHigh, 
+               I64Attr:$levels,
+               DefaultValuedAttr<AutoBroadcastEnumAttr,
+                                "static_cast<int64_t>(MLIRPadType::EXPLICIT)">:$autoBroadcast)>
+{
+  let summary = "Op performing element-wise linear quantization.";
+  let description = [{
+    Input floating point values are quantized into a discrete
+    set of floating point values.
+    
+    Implementation This class creates a node which performs the following
+    operation:
+      round((data - input_low) / (input_high - input_low) * (levels-1)) /
+      (levels-1) * (output_high - output_low) + output_low
+  }];
+
+  let extraClassDeclaration = [{
+    void setLevels(const Attribute& attr) { this->setAttr("levels", attr); }
+  }];
+}
+
+// DepthToSpace Op
+def NGDepthToSpaceOp : 
+    NG_OneResult_Op<"depthToSpace", [NoSideEffect, DeclareOpInterfaceMethods<FusedOp>]>,
+    Arguments<(ins NG_TensorType:$data, 
+               DefaultValuedAttr<I64Attr, "1">:$blockSize, 
+               DepthSpaceModeEnumAttr:$mode)>
```

Comment:
There's a definition where the `mode` is a string. will this work in that case too?